admin_manual/installation/examples/lets_encrypt/apache.rst

```diff
@@ -0,0 +1,149 @@
+Setup Apache
+============
+
+The following is an example setup process for Apache, please adopt to your exact needs.
+
+Apache SSLOpenSSLConfCmd DHParameters
+-------------------------------------
+
+For Apache 2.4.8 or later and OpenSSL 1.0.2 or later, you can generate and specify your `Diffie-Hellman`_ (DH) params file.
+If not already present, add an `SSLOpenSSLConfCmd`_ directive and a new certificate with stronger keys (which improves `forward secrecy`_).
+The OpenSSL command may take a quite a while to complete, so please be patient.
+
+You can place the certificate into any directory of your choice.
+We recommend ``/etc/apache2/`` in this guide, just for sakes of simplicity.
+
+::
+
+  sudo openssl dhparam -out /etc/apache2/dh4096.pem 4096
+  
+Add the following directive to your common SSL configuration:
+
+.. code-block:: apacheconf
+
+  SSLOpenSSLConfCmd DHParameters /etc/apache2/dh4096.pem
+  
+Prepare a virtualHost directive for port 443
+--------------------------------------------
+
+It is easiest, if you create a separate file for the following ``SSL`` directives. 
+If these directives already exist in this Virtual Host, delete them and include the file instead. 
+When the certificate has been created, you can use this file in any SSL Virtual Host for which the certificate is valid, without 
+reissuing.
+
+::
+
+  cd /etc/apache2/
+  sudo mkdir ssl_rules
+  
+Create a file named ``ssl_mydom.tld`` in the newly created directory. 
+
+.. code-block:: apacheconf
+
+   # ssl rules for mydom.tld
+   # eases letsencrypt initial cert issuing
+
+   SSLEngine on
+
+   SSLCertificateChainFile  /etc/letsencrypt/live/mydom.tld/fullchain.pem
+   SSLCertificateKeyFile    /etc/letsencrypt/live/mydom.tld/privkey.pem
+   SSLCertificateFile       /etc/letsencrypt/live/mydom.tld/cert.pem
+
+To reduce the SSL performance penalty, we recommend you use the `SSLUseStapling`_ and `SSLStaplingCache`_ directives:
+Here’s an example configuration:
+
+::
+
+   SSLUseStapling on
+   SSLStaplingCache         shmcb:/tmp/stapling_cache(2097152)
+
```

Comment:
You have killed the ``/.well-known/acme-challenge`` part for port 80. See my first commit. 
**Mandatory, will not work without...**

Reply:
Though it did. I'll check again however, just to be sure.

Reply:
The latest changes re-introduce it.